package.json

```diff
@@ -20,15 +20,18 @@
     "@typescript-eslint/eslint-plugin": "8.8.0",
     "@typescript-eslint/parser": "8.8.0",
     "concurrently": "9.0.1",
+    "dotenv": "^16.4.5",
```

Comment:
faltou deixar as versões exatas para as novas dependências

Reply:
Beleza!, vou pesquisar por um meio de sempre que for instalar uma dependência ela automaticamente sempre ser sua versão exata, acredito que já tava no caminho com o npmrc, onde conseguimos configurar o npm,agora vou procurar a configuração que ele recebe!

Reply:
Achei o save-exact=true !, ele faz com que ao instalar as depêndencias,elas sejam instaladas com sua versão exata